lib/v2/aijishu/index.js

```diff
@@ -0,0 +1,26 @@
+const utils = require('./utils');
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const { type, name = 'newest' } = ctx.params;
+    const u = name === 'newest' ? `https://aijishu.com/` : `https://aijishu.com/${type}/${name}`;
+    const html = await got(u);
+
+    const $ = cheerio.load(html.data);
+    const title = $('title').text();
+    const api_path = $('li[data-js-stream-load-more]').attr('data-api-url');
+
+    const channel_url = `https://aijishu.com/${api_path}?page=1`;
```

Comment:
```suggestion
    const channel_url = `https://aijishu.com${api_path}?page=1`;
```
Duplicated `/` in URL
log: https://github.com/DIYgod/RSSHub/actions/runs/4283984511/jobs/7460302493#step:11:16